apps/web/src/components/layout/components/v2/BridgeUpdateModal.tsx

```diff
@@ -82,7 +82,7 @@ export const BridgeUpdateModal: FC<BridgeUpdateModalProps> = ({ isOpen, toggleOp
 
       await storeInProperLocation(url);
       track('Update endpoint clicked - [Bridge Modal]');
-      successMessage('You have successfuly updated your Novu Endpoint configuration');
+      successMessage('You have successfully updated your Novu Endpoint configuration');
```

Comment:
<!-- Praises highlight something positive. Try to leave at least one of these comments per review. Do not leave false praise (which can actually be damaging). Do look for something to sincerely praise.  -->
👏 **praise:** Nice catch, thank you!‏